test/metrics/test_metrics_general.py

```diff
@@ -66,16 +65,12 @@ def get_disagreements(inputs, bins, counts, test):
 @pytest.mark.parametrize("input_shape", [(1, 72, 144), (1, 360, 720)])
 def test_histogram(device, input_shape, rtol: float = 1e-3, atol: float = 1e-3):
     DistributedManager._shared_state = {}
-    if (device == "cuda:0") and (not dist.is_initialized()):
+    if (device == "cuda:0") and (not DistributedManager.is_initialized()):
```

Comment:
I am not sure what is best practices here, you can also compare device.type against "cuda" and device.index against 0.

Reply:
I think in this particular example, comparing against the string is better because it is coming from the pytest parameterize above and so is just a string and not a `torch.device` object.